### PR TITLE
Fixing default value for region not applied when the user inserts whitespace in the answer.

### PR DESCRIPTION
**Pull request by MatheusClaudi**

### CODE REVIEW

**Review by Vieira-Carlos — APPROVED** [review]

File: ml_git/commands/wizard.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def wizard_for_field(context, field, input_message, required=False, wizard_flag=
     else:
         try:
             new_field = check_empty_for_none(request_new_value(input_message, required, type))
-            if not new_field:
+            if not new_field or new_field.isspace():
                 return default
             return new_field
```
Comment (lucasncabral) [inline ml_git/commands/wizard.py:49]: To avoid possible problems, we can trim the value informed before returning it.

Comment (MatheusClaudi) [reply ml_git/commands/wizard.py:49]: About the tests, I will keep that in mind, and about the trim, do you mean removing all external whitespaces from the string and modify to comparate with an empty string in line 47?

Comment (lucasncabral) [reply ml_git/commands/wizard.py:49]: No, just remove the outer spaces the user has passed in the field. In line 49.

```suggestion
            return new_field.strip()
```

Comment (MatheusClaudi) [reply ml_git/commands/wizard.py:49]: Changes have been made

**Review by lucasncabral — APPROVED** [review]

**Pull request merged**